contracts/RiskFund/RiskFund.sol

```diff
@@ -203,9 +200,17 @@ contract RiskFund is
     function transferReserveForAuction(address comptroller, uint256 amount) external override returns (uint256) {
         address shortfall_ = shortfall;
         require(msg.sender == shortfall_, "Risk fund: Only callable by Shortfall contract");
-        require(amount <= poolReserves[comptroller], "Risk Fund: Insufficient pool reserve.");
+        require(
+            amount <= poolsAssetsReserves[comptroller][convertibleBaseAsset],
+            "Risk Fund: Insufficient pool reserve."
+        );
         unchecked {
-            poolReserves[comptroller] = poolReserves[comptroller] - amount;
+            poolsAssetsReserves[comptroller][convertibleBaseAsset] =
+                poolsAssetsReserves[comptroller][convertibleBaseAsset] -
+                amount;
+        }
+        unchecked {
+            assetsReserves[convertibleBaseAsset] = assetsReserves[convertibleBaseAsset] - amount;
         }
```

Comment:
Can we prove it's safe to do unchecked here?

Reply:
As we see the mapping `poolsAssetsReserves` and `assetsReserves` are updated simultaneously everywhere in the contract and we are checking [here](https://github.com/VenusProtocol/isolated-pools/blob/3427b24382f6a59d8c132701afe9c2bd0c62d881/contracts/RiskFund/RiskFund.sol#L204) the `amount` is less than the `poolsAssetsReserves[comptroller][convertibleBaseAsset]`